### DISCUSSION TITLE
I wish there was a hotkey option.

### DISCUSSION THREAD

**Discussion by kksshh231** [BazzaCuda/CustomMenu#14] [Q&A] [answered]
I wish there was an option to change the hotkey.
It currently works with right mouse button, but I wish it worked with ctrl + right mouse button.
So I wish there was an option to specify a separate hotkey.

**Comment by BazzaCuda** [comment]
Hi, that's a very good idea...allowing the user to have right-click bring up the standard desktop menu and Ctrl-click bring up CustomMenu, if that's what they prefer.

I'm working on fixing another issue at the moment, but as soon as I've done that I'll work out a very quick way of allowing what you've suggested. 

Edit: this option will be in the next release, due tomorrow.

**Reply by BazzaCuda** [reply]
The ability to activate **CustomMenu** when you _CTRL_-rightclick the desktop (and for the Windows menu to activate when you simply right-click the desktop without the CTRL key) is in today's [latest release](https://github.com/BazzaCuda/CustomMenu/releases/tag/v1.0.5)

Thanks for the suggestion.

**Reply by kksshh231** [reply]
Thank you so much for the quick feedback. I don't know how to say this...
Thank you so much ^^

**Reply by kksshh231** [reply]
I found that there was one problem when creating and running the ctrl-click file.
Shift + right mouse button click does not quit the program.
I think I need to look into this part.

**Reply by kksshh231** [reply]
When running CustomMenu on the desktop, the context menu does not disappear.
There is a problem. To make it disappear, just click on anything on the desktop and it will disappear.

**Reply by BazzaCuda** [reply]
This is all covered in the release notes. Please read the section marked "EXPERIMENTAL", https://github.com/BazzaCuda/CustomMenu/releases/tag/v1.0.5

**Comment by CoolHandDevil505** [comment]
In the same kind of ideas, would it be possible to have CustomMenu activation by a keyboard shortcut, available from anywhere?

**Reply by rungek** [reply]
I 100% agree. I almost never have my desktop visible, so I need to show it before I can access the Menu. A shortcut from anywhere, desktop or not, would save clicks.
Great product!

**Reply by BazzaCuda** [reply]
Done. Just preparing the release now ;)

**Reply by BazzaCuda** [reply, marked as answer]
Hotkey activation: https://github.com/BazzaCuda/CustomMenu/releases/tag/v1.0.6
